Homework/Duc_Trinh/217-contains-duplicate.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+ var containsDuplicate = function(nums) {
+    const hashmap = {}
+    let res = false
+    for (let i = 0; i < nums.length; i++) {
```

Comment:
a hashmap from int to bool should be enough?

Reply:
@quangh33 right, I have fixed it. Please take a look